spec/01-unit/02-conf_loader_spec.lua

```diff
@@ -262,6 +262,19 @@ describe("Configuration loader", function()
       assert.is_nil(err)
       assert.is_table(conf)
     end)
+    it("errors when hosts have a bad format in cassandra_contact_points", function()
+        local conf, err = conf_loader(nil, {
+            cassandra_contact_points = "some/really\\bad/host\name,addr2"
+        })
+        assert.is_nil(conf)
```

Comment:
the second `\` also needs escaping, as well as testing for a proper error message, as I think it currently errors with something like "bad name" without any 'contact_points' context